util/log/file.go

```diff
@@ -85,36 +97,86 @@ func shortHostname(hostname string) string {
 	return hostname
 }
 
-// logName returns a new log file name containing tag, with start time t, and
-// the name for the symlink for tag.
-func logName(tag string, t time.Time) (name, link string) {
-	name = fmt.Sprintf("%s.%s.%s.log.%s.%04d%02d%02d-%02d%02d%02d.%d",
-		program,
-		host,
-		userName,
-		tag,
-		t.Year(),
-		t.Month(),
-		t.Day(),
-		t.Hour(),
-		t.Minute(),
-		t.Second(),
+// removePeriods removes all extraneous periods. This is required to ensure that
+// the only periods in the filename are the ones added by logName so it can
+// be easily parsed.
+func removePeriods(s string) string {
+	return strings.Replace(s, ".", "", -1)
+}
+
+// logName returns a new log file name containing the severity, with start time
+// t, and the name for the symlink for the severity.
+func logName(severity Severity, t time.Time) (name, link string) {
+	// Replace the ':'s in the time format with '_'s to allow for log files in
+	// Windows.
+	tFormatted := strings.Replace(t.Format(time.RFC3339), ":", "_", -1)
```

Comment:
I think I would extract this into a method (which can have a nice descriptive name like `escapeForWindows`)